package.json

```diff
@@ -96,5 +95,15 @@
     "shelljs": "^0.8.4",
     "ts-node": "^8.10.2",
     "typescript": "~4.8.2"
+  },
+  "pnpm": {
+    "packageExtensions": {
+      "@nrwl/devkit": {
```

Comment:
Do you know where nrwl/devkit is coming from? That looks unexpected

Reply:
Looks like it is no longer being used, see yarn.lock in https://github.com/angular/material.angular.io/commit/ac0acaebc46e3c29e09fa36af3d323ed049574b2#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deL3703